azure-iot-device/tests/common/pipeline/config_test.py

```diff
@@ -0,0 +1,230 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import pytest
+import abc
+import six
+from azure.iot.device import ProxyOptions
+
+
+@six.add_metaclass(abc.ABCMeta)
+class PipelineConfigInstantiationTestBase(object):
```

Comment:
>PipelineConfigInstantiationTestBase [](start = 6, length = 35)

<eyes crossing at 7 AM> Man, this "InstantiationTestBase" convention makes for some long identifiers.  No change requested.  Just murmuring to myself.  Get off my lawn.

Reply:
Some of these test class names could be confused for parody of Java tbh :joy: 

---
In reply to: [429295058](https://github.com/Azure/azure-iot-sdk-python/pull/555#discussion_r429295058) [](ancestors = 429295058)